examples/kebechet/alerting.md

```diff
@@ -0,0 +1,25 @@
+# Alerting
+
+Once we have all metrics collected and stored, we can define the alerts that are going to be triggered. You can get notified of these alerts via different methods:
+
+- Github Issues, using [Prometheus GitHub AlertManager](https://github.com/m-lab/alertmanager-github-receiver), a webhook receiver that creates GitHub issues from alerts. To learn more about setting up the GitHub Alertmanager you can refer to our [documentation](https://www.operate-first.cloud/operations/sre/incident-management/github-receiver-setup.md).
+
```

Comment:
did we still want to include the other methods i.e via email and GChat notification by Thoth bots?